packages/workflows/src/Create.ts

```diff
@@ -96,31 +99,86 @@ export class CreateWorkflow{
             accessType: AccessType,
             deleteCompletedJobs: DeleteCompletedJobs
         };
-        if (!isNullOrUndefined(VariableInputFile)){
+        if (!isNullOrUndefined(VariableInputFile)) {
             if (VariableInputFile.charAt(0) === "/" && VariableInputFile.charAt(1) === "/") {
-            VariableInputFile = VariableInputFile.substring(1);
-        }
+                VariableInputFile = VariableInputFile.substring(1);
+            }
             data.variableInputFile = VariableInputFile;
         }
-        if (!isNullOrUndefined(Variables)){
+        if (!isNullOrUndefined(Variables)) {
             data.variables = this.parseProperties(Variables);
         }
-        if (isNullOrUndefined(AssignToOwner)){
+        if (isNullOrUndefined(AssignToOwner)) {
             data.assignToOwner = true;
         }
-        if (isNullOrUndefined(AccessType)){
+        if (isNullOrUndefined(AccessType)) {
             data.accessType = "Public";
         }
-        if (isNullOrUndefined(DeleteCompletedJobs)){
+        if (isNullOrUndefined(DeleteCompletedJobs)) {
             data.deleteCompletedJobs = false;
         }
 
         const resourcesQuery: string = `${WorkflowConstants.RESOURCE}/${zOSMFVersion}/${WorkflowConstants.WORKFLOW_RESOURCE}`;
 
         return ZosmfRestClient.postExpectJSON<ICreatedWorkflow>(session, resourcesQuery, [Headers.APPLICATION_JSON], data);
```

Comment:
Should the `createWorkflow` and `createWorkflowLocal` create a data object and call their respective `version2` functions instead of revalidating everything?